3-async-await/solution.js

```diff
@@ -20,16 +20,74 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
+const getPrices = require('./prices');
+const getProducts = require('./products');
 
-    // You generate your id value here
+async function solution() {
+  // YOUR SOLUTION GOES HERE
+  // const productObject = {};
+  // You generate your id value here
+  const dateID = Date.now();
+  const dateIdArray = dateID.toString().split('');
+  const idProduct = dateIdArray
+    .slice(dateIdArray.length - 2, dateIdArray.length)
+    .join('');
+  //function to create a obj
+  const objectCreatorHandler = (productResponse, description) => {
+    productObject = {
```

Comment:
I think you want a const here right? Otherwise it might do something unexpected

Reply:
Yes hahaa i was checking and i believe i delete const before pushing the changes, sorry about it, i'm going to pay atention before commit any change